src/linear_algebra/basic.lean

```diff
@@ -2458,17 +2521,6 @@ quotient_inf_equiv_sup_quotient_symm_apply_eq_zero_iff.2 hx
 
 end isomorphism_laws
 
-section prod
-
-lemma is_linear_map_prod_iso {R M M₂ M₃ : Type*}
-  [comm_semiring R] [add_comm_monoid M] [add_comm_monoid M₂]
-  [add_comm_group M₃] [semimodule R M] [semimodule R M₂] [semimodule R M₃] :
-  is_linear_map R (λ(p : (M →ₗ[R] M₂) × (M →ₗ[R] M₃)),
-    (linear_map.prod p.1 p.2 : (M →ₗ[R] (M₂ × M₃)))) :=
-⟨λu v, rfl, λc u, rfl⟩
-
-end prod
-
```

Comment:
This was added back in #966:

https://github.com/leanprover-community/mathlib/commit/ade99c8ca957b815760ff01b0ac6897b994a15ba#diff-923fd2b8078cd05fe5d03aee47329529e737233260a69f86464faf2602c94ccbR1303-R1306

It's not used anywhere, and is now just `(prod_equiv R).is_linear_map` or similar.